ktor-client/ktor-client-core/posix/src/io/ktor/client/utils/CoroutineUtilsPosix.kt

```diff
@@ -15,3 +16,16 @@ public actual fun Dispatchers.clientDispatcher(
     threadCount: Int,
     dispatcherName: String
 ): CoroutineDispatcher = Unconfined
+
+internal actual fun checkCoroutinesVersion() {
+    try {
+        val parent = Job()
+        parent.freeze()
+        Job(parent)
+    } catch (cause: Throwable) {
+        val message = "Ktor native HttpClient requires kotlinx.coroutines version with `native-mt` suffix" +
+            "(like `1.3.9-native-mt`). Consider checking the dependencies."
```

Comment:
space between word "suffix" and opening "("

Reply:
LGTM